### PR TITLE
chore: update portal loop archiver version

### PR DESCRIPTION
**Pull request by zivkovicmilos**
## Description

This PR updates the portal loop tx-archiver version to `v0.4.2`, which eliminates the millisecond timestamp issue

### CODE REVIEW

**Comment by Gno2D2** [comment]
I'm a bot that assists the Gno Core team in maintaining this repository. My role is to ensure that contributors understand and follow our guidelines, helping to streamline the development process.

The following requirements must be fulfilled before a pull request can be merged.
Some requirement checks are automated and can be verified by the CI, while others need manual verification by a staff member.

These requirements are defined in this [configuration file](https://github.com/gnolang/gno/tree/master/contribs/github-bot/internal/config/config.go).

## Automated Checks

 🟢 The pull request head branch must be up-to-date with its base ([more info](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch))


## Manual Checks

*No manual checks match this pull request.*

<details><summary><b>Debug</b></summary><blockquote>
<details><summary><b>Automated Checks</b></summary><blockquote>

<details><summary>The pull request head branch must be up-to-date with its base (more info)</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Then
```
🟢 Requirement satisfied
└── 🟢 Head branch (dev/zivkovicmilos/pl-archiver) is up to date with base (master): behind by 0 / ahead by 1

```
</blockquote></details>

</blockquote></details>



</blockquote></details>

**Review by sw360cab — APPROVED** [review]

**Review by sw360cab — APPROVED** [review]

**Pull request merged**
